swap_meet/decor.py

```diff
@@ -1,2 +1,6 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Decor(Item):
+    def __init__(self, age=None, condition=None, category="Decor"):
```

Comment:
Same comment here as above in Clothing class, let's directly pass "Decor" into the super class constructor and remove it from the available initializer parameters.